src/widgets/splits/SplitInput.cpp

```diff
@@ -76,6 +76,13 @@ SplitInput::SplitInput(QWidget *parent, Split *_chatWidget,
                                        });
 }
 
+SplitInput::~SplitInput()
+{
+    // shortcuts are registered on the parent widget,
+    // but capture this widget.
+    this->clearShortcuts();
```

Comment:
Is this change still relevant?

Reply:
It's still wrong, but a `SplitInput` is never destroyed without the parent (reply-thread or split).